go/plugins/dotprompt/dotprompt.go

```diff
@@ -321,3 +329,99 @@ func sortSchemaSlices(s *jsonschema.Schema) {
 		sortSchemaSlices(s.Items)
 	}
 }
+
+// Adds a variant name to the prompt
+func WithVariant(variant string) PromptOption {
+	return func(p *Prompt) error {
+		p.Config.Variant = variant
+		return nil
+	}
+}
+
+// Adds tools to the prompt
```

Comment:
I believe proper comment format would be "WithTools adds tools to the prompt." Same for others.